tests/unittests/test_main.py

```diff
@@ -101,21 +105,22 @@ def test_no_form_given(self, mock_forms):
         Test when no forms are given in config, an error is raised with an expected message.
         """
         config = {}
-        with self.assertRaises(NoFormsProvidedError) as e:
+        with self.assertLogs(level='INFO') as log_statement:

```

Comment:
we should also add assertions around ensuring that api_forms is called / returned as expected, even if we mock the responses.

Reply:
added